src/assets/scss/custom.scss

```diff
@@ -1,9 +1,9 @@
-$primary: #404F65;
-$secondary: #D65F30;
-$success: #219653;
-$info: #3144EC;
-$light: #CFDBE1;
-$warning: #E2B93B;
-$error: #EB5757;
+$primary: #00393f;
+$secondary: #e84614;
```

Comment:
Adding here, because I cant find the location, The favicon is visible in Chrome but not in Firefox

Reply:
The favicon should me much smaller, then it will also load faster - see [here](https://www.emergeinteractive.com/insights/detail/the-essentials-of-favicons/).

Reply:
Fixed. 192x192 now